VespaGAS/Models/Gasoline.swift

```diff
@@ -1 +1,11 @@
-typealias Gasoline = Volume
+import Foundation
+
+struct Gasoline: Volume {
+    var amount: Float
```

Comment:
Good spot! I'm going to change it even if I would like to remove them. 
As I said in the comment at the beginning of the commit, I would like to remove these 2 lines as they are only there to make the compiler happy. Do you have any idea?